README.md

```diff
@@ -7,7 +7,7 @@
 
 ----
 
-Check out [vella.ai/auth](https://vella.ai/auth/) for a demo app using this library for local-only authentication with WebAuthn and local encryption.
+Check out the library tests for a demo of local-only authentication with WebAuthn and local encryption:
```

Comment:
Bummer that Vella removed that demo, but... we can just remove this sentence entirely instead of duplicating the tests link.

Reply:
sounds good. my first revision removed the link from that sentence and updated the text to serve as a way to introduce the tests, but i’m happy to just remove the sentence entirely.